tests/test_cookiecutter_python_library.py

```diff
@@ -32,7 +32,7 @@
     "author_email": "cookie@monster.org",
     "author_name": "Cookie Monster",
     "library_name": "cookie_lover",
-    "github_org": "bakery_org",
+    "github_org": "openedx",
```

Comment:
I infer that you've made this change to test the new branch, and that the other cookiecutter tests use non-openedx orgs?

Reply:
Yes, this was to have a mixture of organizations across all of the tests.